packages/office-ui-fabric-react/src/components/Label/LabelMixins.scss

```diff
@@ -15,4 +15,15 @@
     color: $ms-color-error;
     @include ms-padding-right(12px);
   }
+}
+
+@mixin ms-Icon-is-required {
+  &::before {
+    content: ' *';
+    color: $ms-color-error;
+    position: absolute;
+    top: -4px;
+    @include ms-right(8px);
+    @include ms-padding-right(12px);
```

Comment:
this is very specific to the textfield scenario no? Why not just define it in textfield? Mixins like this make things messy. Especially now that TextField.scss have an implicit dependency on LabelMixins.scss.